databricks/koalas/frame.py

```diff
@@ -878,6 +878,85 @@ def iteritems(self):
         cols = list(self.columns)
         return list((col_name, self[col_name]) for col_name in cols)
 
+    def to_csv(self, path_or_buf=None, sep=',', na_rep='', columns=None, header=True,
```

Comment:
@RainFung, this is implemented under `_Frame.py` in `general.py`. You can remove that implementation or move those implementation to there.